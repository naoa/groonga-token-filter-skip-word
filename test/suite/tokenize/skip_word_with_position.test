register token_filters/skip_word

table_create skip_word_with_positions TABLE_HASH_KEY ShortText
load --table skip_word_with_positions
[
{"_key": "and"}
]

tokenize TokenBigram "Hello and World" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterSkipWordWithPosition

