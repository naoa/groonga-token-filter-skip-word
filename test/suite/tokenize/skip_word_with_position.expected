register token_filters/skip_word
[[0,0.0,0.0],true]
table_create skip_word_with_positions TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table skip_word_with_positions
[
{"_key": "and"}
]
[[0,0.0,0.0],1]
tokenize TokenBigram "Hello and World"   --normalizer NormalizerAuto   --token_filters TokenFilterSkipWordWithPosition
[[0,0.0,0.0],[{"value":"hello","position":0},{"value":"world","position":1}]]
